libraries/AP_HAL_ChibiOS/hwdef/KakuteF7fix/hwdef-bl.dat

```diff
@@ -0,0 +1,48 @@
+# hw definition file for processing by chibios_pins.py
```

Comment:
what is this KakuteF7fix build? Did you accidentally commit it perhaps?

Reply:
it was not supposed to be there. sorry. i will look at it all on monday